polaris-tokens/src/token-groups/shadow.ts

```diff
@@ -1,7 +1,17 @@
 import type {MetadataProperties, Experimental} from '../types';
 
 type ShadowAliasExperimental = Experimental<
-  'card-sm' | 'card-md' | 'card-lg' | 'button' | 'button-disabled'
+  | 'card-sm'
+  | 'card-md'
+  | 'card-lg'
+  | 'button'
+  | 'button-disabled'
+  | 'inset-button-primary'
+  | 'button-primary-strong-hover'
+  | 'button-primary-strong'
+  | 'button-primary'
+  | 'button-primary-hover'
+  | 'inset-button'
```

Comment:
As you may know, we haven't established naming conventions for component-specific Polaris tokens yet, as some team members had strong opinions that they don't belong in the tokens library. That's why we've been using `--pc-*` prefixed variables. However, since we are making compromises with experimental tokens, I think there's an opportunity to establish consistency here.

My proposal is to use the same pattern we currently have for general Polaris tokens:
`--p-<token-group>-<token-name>`

For component-specific tokens, we can extend this pattern to:
`--p-<token-group>-<component>-<token-name>`

Most of the introduced tokens already follow this pattern, with the exception of `--p-shadow-inset-button` and `--p-shadow-inset-button-primary`, which I suggest renaming to `--p-shadow-button-inset` and `--p-shadow-button-inset-primary`, respectively.

Thoughts?

Reply:
Update looks good! Note: I use `<token-name>` as a catch all for whatever token naming convention design establishes as each token group is very nuanced (for example [Color token naming](https://polaris.shopify.com/whats-new/version-11-color#token-naming)). That said, your updates are align with the original proposal of `--p-<token-group>-<component>-<token-name>` 👍 Thanks!